### DISCUSSION TITLE
Updating Yosys in OpenROAD flow

### DISCUSSION THREAD

**Discussion by ishwars1618** [The-OpenROAD-Project/OpenROAD-flow-scripts#1065] [Q&A]
Can anyone list the steps needed to update Yosys to the latest version in the OpenROAD distribution?

**Comment by vijayank88** [comment]
Install latest yosys tool. Update this [env.sh](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/blob/df9a44242f8474591b0dcdd19c67f059b3d7eebe/env.sh#L15) locally to source yosys as per your need.
